### PR TITLE
Add Cirrus CI config file

### LINKED ISSUE #3: Add CI to run tests
It is extremly usefull :)

### PR DESCRIPTION
**Pull request by AlexWayfer**
Resolve #3

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+aliases:
+  - &bundle_cache
```
Comment (fkorotkov) [inline .cirrus.yml]: I think the syntax should be like this:

```suggestion
  bundle_cache: &bundle_cache |
```

Comment (AlexWayfer) [reply .cirrus.yml]: Why (for what) `|`?

Comment (AlexWayfer) [reply .cirrus.yml]: Still no…

Comment (AlexWayfer) [reply .cirrus.yml]: `|` for multiline strings, as I know.

Aliases are correct, and parsed (in Ruby) correctly.

File: .cirrus.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+aliases:
+  bundle_cache: &bundle_cache |
```
Comment (fkorotkov) [inline .cirrus.yml]: My bad. `|` is redundant. Cirrus CI is using SnakeYAML for parsing and this is the [format it supports](https://bitbucket.org/asomov/snakeyaml/wiki/Documentation#markdown-header-aliases). Maybe YAML standard allows several forms. 🤔 

```suggestion
  bundle_cache: &bundle_cache
```

File: .cirrus.yml (hunk 1 above)
Comment (AlexWayfer) [reply .cirrus.yml]: You can check YAML file here, for example: https://yaml-online-parser.appspot.com/

File: .cirrus.yml (hunk 2 above)
Comment (AlexWayfer) [reply .cirrus.yml]: I tried this, and no: https://cirrus-ci.com/build/5926221328154624

Comment (AlexWayfer) [reply .cirrus.yml]: > Maybe YAML standard allows several forms. :thinking:

According to the official reference, no.

Comment (fkorotkov) [reply .cirrus.yml]: Found an issue in SnakeYML, implemented a workaround, should be deployed in ~20-30 minutes. Will ping you when it's available. Thanks for reporting and sorry for the inconvenience.

Comment (fkorotkov) [reply .cirrus.yml]: This should work now. You can try to re-trigger your build here: https://cirrus-ci.com/build/5926221328154624

Comment (AlexWayfer) [reply .cirrus.yml]: Thank you! I glad to find the issue.

**Pull request merged**